ui/package.json

```diff
@@ -1,37 +1,58 @@
 {
-  "name": "feast-ui",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
+  "name": "@feast-dev/feast-ui",
+  "version": "0.1.1",
+  "private": false,
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/feast-ui.cjs",
+  "module": "./dist/feast-ui.module.js",
+  "peerDependencies": {
     "@elastic/datemath": "^5.0.3",
-    "@elastic/eui": "^50.0.0",
-    "@emotion/react": "^11.8.1",
-    "@testing-library/jest-dom": "^5.16.2",
-    "@testing-library/react": "^12.1.3",
-    "@testing-library/user-event": "^13.5.0",
+    "@elastic/eui": "^46.1.0",
```

Comment:
any reason why we're using some old versions?

Reply:
This is simply the version we started with during the hackathon. I'm not sure what is the right thing to do however, b/c:

1. On the one hand, EUI is a peer dependency, which means that customers could potentially use different versions, and we want our package to play nice with others.
2. On the other hand ... new updates to EUI might break our module. 

I'm not really sure what the best practice here is.